lib/fs.js

```diff
@@ -1838,7 +1842,9 @@ if (isWindows) {
   nextPart = function nextPart(p, i) {
     for (; i < p.length; ++i) {
       const ch = p.charCodeAt(i);
-      if (ch === 92/*'\'*/ || ch === 47/*'/'*/)
+
+      // if separator
```

Comment:
Nit: could this be changed so something like "Check for a separator character"?

And please always begin comments with a capital letter :-)

Reply:
@BridgeAR ok, sorry

Reply:
@BridgeAR done